# TODO: Remove this file - Uploaded just as a sample failure log showcasing pytest params and diff.

data = [2], data2 = [1]

    @pytest.mark.parametrize("data",[
        [1],
        [2]
    ])
    @pytest.mark.parametrize("data2",[
        [1],
        [2]
    ])
    def test_are_equal_custom(data, data2):
>       assert data == data2
E       assert [2] == [1]
E         At index 0 diff: 2 != 1
E         Use -v to get the full diff

python/cudf/cudf/fuzz_tests/test_parquet.py:17: AssertionError
data = [1], data2 = [2]

    @pytest.mark.parametrize("data",[
        [1],
        [2]
    ])
    @pytest.mark.parametrize("data2",[
        [1],
        [2]
    ])
    def test_are_equal_custom(data, data2):
>       assert data == data2
E       assert [1] == [2]
E         At index 0 diff: 1 != 2
E         Use -v to get the full diff

python/cudf/cudf/fuzz_tests/test_parquet.py:17: AssertionError
